src/layer/vector/SVG.js

```diff
@@ -104,6 +104,11 @@ export var SVG = Renderer.extend({
 			DomUtil.addClass(path, 'leaflet-interactive');
 		}
 
+		if (layer.options.keyboard) {
+			path.tabIndex = '0';
+			path.setAttribute('role', 'img');
```

Comment:
SVG `path` is [implicitly](https://www.w3.org/TR/SVG2/struct.html#:~:text=%E2%80%98path%E2%80%99-,graphics%2Dsymbol%20role,-if%20the%20element) [`role="graphics-symbol"`](https://www.w3.org/TR/graphics-aria-1.0/#graphics-symbol) (albeit only [if `aria-label`, `aria-labelledby`, or `title` is used](https://www.w3.org/TR/SVG2/struct.html#TermInclusionCriteria)), I don't think there's a need to override it.

Reply:
Agree, now after I added `<title>`, it is unnecessary and gone.